package.json

```diff
@@ -21,19 +21,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "go-ipfs": "0.3.6",
-    "ipfs-api": "2.2.1",
-    "kew": "0.5.0",
-    "lodash": "3.6.0",
+    "go-ipfs": "0.3.7-3",
```

Comment:
do we need to be specific? can we not say `0.3.7` and get the (only) one that's published? i checked this manually with `npm install 0.3.7` and it installed one... but not with a dep)